docs/usage/appsets-in-any-namespace.md

```diff
@@ -0,0 +1,77 @@
+# ApplicationSets in Any Namespace
+
+**Current feature state**: Beta
+
+!!! note
+    This feature is considered beta feature in upstream Argo CD as of now. Some of the implementation details may change over the course of time until it is promoted to a stable status.
+
+Argo CD supports managing `ApplicationSet` resources in namespaces other than the control plane's namespace. Argo CD administrators can define a certain set of namespaces where `ApplicationSet` resources may be created, updated and reconciled in. 
+
+In order to manage `ApplicationSet` outside the Argo CD's control plane namespace, two prerequisites must be satisfied:
+
+1. The Argo CD should be cluster-scoped
+2. The enabled namespace should be entirely covered by the [Apps in Any Namespace](./apps-in-any-namespace.md)
+
+## Enable ApplicationSets in a namespace
+
+To enable this feature in a namespace, add the namespace name under `.spec.applicationSet.sourceNamespaces` field in ArgoCD CR.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+spec:
+  applicationSet:
+    sourceNamespaces:
+      - some-namespace
+```
+
+As of now, wildcards are not supported in `.spec.applicationSet.sourceNamespaces`. 
+
+!!! important 
+    Ensure that [Apps in Any Namespace](./apps-in-any-namespace.md) is enabled on target namespace i.e the target namespace name is covered under `.spec.sourceNamespaces` in ArgoCD CR.
+
+The operator creates/modifies below RBAC resources when ApplicationSets in Any Namespace is enabled
+
+|Name|Kind|Purpose|
+|:-|:-|:-|
+|`<argoCDName-argoCDNamespace>-argocd-applicationset-controller`|ClusteRole & ClusterRoleBinding|for `applicationset-controller` to watch & list `ApplicationSets` at cluster-level|
```

Comment:
@svghadi Do you mean this - For `ApplicationSet` controller to watch and list `ApplicationSet` resources at cluster-level?